examples/example1d_SAM_ParametricStudy/README.md

```diff
@@ -0,0 +1,19 @@
+# Example1d - MOOSE/SAM Plugin
+
+## Purpose
+
+This example provides a demonstration on how to use WATTS to perform multiple runs and save selected outputs from all runs to a single '.csv' file for ease of comparison, post-processing, and visualization.
+
+## Code(s)
+<ul>  
+<li>SAM</li> 
+</ul>
+
+## Keywords
+<ul>  
+<li>Parametric study</li>
+<li>Results extraction</li>
+<li>Post-processing</li>
+</ul>
+
+## Notes
```

Comment:
Were you planning on adding something below here? If not, I'd probably just get rid of this heading